drape_frontend/user_event_stream.hpp

```diff
@@ -72,7 +72,8 @@ class TouchEvent : public UserEvent
     TOUCH_DOWN,
     TOUCH_MOVE,
     TOUCH_UP,
-    TOUCH_CANCEL
+    TOUCH_CANCEL,
+    TOUCH_NONE
```

Comment:
nit: I'd prefer to put NONE on the first place.

Reply:
Done